gen/feat_metadata.tmpl

```diff
@@ -0,0 +1,83 @@
+package iotago
+
+// Code generated by go generate; DO NOT EDIT. Check gen/ directory instead.
+
+import (
+	"bytes"
+
+	"github.com/iotaledger/hive.go/lo"
+	"github.com/iotaledger/hive.go/serializer/v2"
+)
+
+type {{.Name}}Entries map[string][]byte
```

Comment:
There is no mention whatsoever about ordering of the map. This makes serialization non-deterministic.

Reply:
This is now also forcing the use of a key everytime. (e.g. `data` as in every changed testcase)

Reply:
Why is this better than something like IRC-27 or IRC-30? The protocol does not need the map here to function.